github/Contribution.md

```diff
@@ -0,0 +1,98 @@
+# Contribute to IAI repositories
+
+Everyone is welcome to get involved and we appreciate everybody's contribution.
+Code contributions are not the only way to be involved. 
+Proposing new features or improvement, giving feedback, and improving the documentation are also valuable for the improvement of a library.
+
+## Ways to contribute
+
+There are several ways you can contribute:
+
+  * Fix open issues.
+  * Submit new issues related to desired feature, new ideas, or bugs.
+  * Contribute to the documentation.
+
+## Fix open issues
+
+If you have an idea to fix an open issue, feel free to work on it and open a [Pull Request](#create-a-pull-request).
```

Comment:
Might be good to specify that the PR should come from a fork.

Reply:
Done.